tests/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright © 2021 United States Government as represented by the Administrator of the
 # National Aeronautics and Space Administration.  All Rights Reserved.
 
-__all__ = ['test_base_models', 'test_battery', 'test_examples', 'test_centrifugal_pumps', 'test_pneumatic_valve', 'visualize']
```

Comment:
Why are these removed?

Reply:
These were removed because they were never defined. CodeFactor didnt like it. Would you like me to add them back?

Reply:
Nope, you're right. I wonder what happened to them.